docs/tableBeamGroup1.adoc

```diff
@@ -173,7 +173,12 @@ e|Variables | |
  
  |{var}float transducer_gain(ping_time, beam, frequency) |MA |Gain of the transducer beam. This is the parameter that is set from a calibration exercise. Necessary for conversion equation type 1, 3 and 4.
  3+|{attr}:long_name = "Gain of transducer" 
- 3+|{attr}:units = "dB" 
+ 3+|{attr}:units = "dB"
+
+ |{var}float calibrated_frequency(frequency) |M |Frequencies for which transducer gain values have been estimated during the calibration exercise.
```

Comment:
Would this definition be better placed in the Coordinate Variable part of the table? This makes no difference to the variable itself, but makes it clearer that the main use is as a coordinate for another variable. The netcdf convention for coordinate variables is that the variable name is the same as the dimension name, so it would be `float frequency(frequency)`

Reply:
Agreed and corrected (sorry I didn't get your comment on the first time)